### ISSUE TITLE
<br/> not working in Internet Explorer

### ISSUE THREAD

**Issue by dtslvr** [tameemsafi/typewriterjs#23]
The example from https://jsfiddle.net/shbqy0mv/9/ is not working correctly in Internet Explorer 11. 

```
typewriter.typeString('<strong>Human: </strong>')
  .typeString('Hello how are you?')
  .pauseFor(2500)
  .typeString('<br/>')
  .typeString('<strong>Bot: </strong>')
  .typeString('I am fine thanks, how are you?')
  .pauseFor(2500)
  .typeString('<br/>')
  .typeString('<strong>Human: </strong> ')
  .typeString('I am great!')
  .pauseFor(2500)
  .start();
```

`<br/>` tags get lost and all strings are displayed in one line :/ Unfortunately I don't see an error message in the developer tools.

Do you have an idea how to solve it?

**Comment by tameemsafi** [comment]
The issue has now been fixed in `v2.5.0`. Let me know if it is working for you.

**Comment by dtslvr** [comment]
Great, thank you very much for your quick update! It works fine.
